runing/services.py

```
@@ -0,0 +1,41 @@
+        if db_user:
+            raise HTTPException(status_code=400, detail="Email already registered")
+
+        created_user = self.user_repository.create_user(user)
+        return schemas.User.model_validate(created_user)
+
+    def get_user(self, user_id: int) -> schemas.User:
+        db_user = self.user_repository.get_user(user_id)
+        if not db_user:
+            raise HTTPException(status_code=404, detail="User not found")
+
+    def get_user(self, skip: int = 0, limit: int = 100) -> List[schemas.User]:
```

Ruff (0.11.9)

Redefinition of unused `get_user` from line 19

(F811)